### PR TITLE
Tree max

### PR DESCRIPTION
**Pull request by rangelMatt**
- [x] Top-level README “Table of Contents” is updated
 - [x] README for this challenge is complete
 - [x] Summary, Description, Approach & Efficiency, Solution
 - [x] Picture of whiteboard
 - [x] Link to code
 - [x] Feature tasks for this challenge are completed
 - [x] Unit tests written and passing
 - [x] “Happy Path” - Expected outcome
 - [x] Expected failure
 - [x] Edge Case (if applicable/obvious)

### CODE REVIEW

File: python/data_structures/binary_tree.py [hunk 1]
```diff
@@ -69,4 +70,22 @@ def climb(root, values): # root = a Node or more. Recursive function
...
+            value = self.in_order()
+            max_value = value[0]
+
+            for num in value:
+                if num > max_value:
+                    max_value = num
+
+            return max_value
+
+        else:
```
Comment (aaron-imbrock) [inline python/data_structures/binary_tree.py:89]: Better would be 

```
def find_maximux_value(self):
     if self.root is None:
          return None

    # you could then add logic here to handle negative values
```
